.github/workflows/build-node-go.yml

```
@@ -81,19 +79,52 @@ jobs:
+        uses: actions/download-artifact@v4
+        with:
+          name: xmtp-prune-mls-digest
+          path: digests
+
+      - name: Read and export digests
+        id: digests
+        run: |
+            set -euo pipefail
+            NODE_GO_DIGEST=$(cut -d= -f2 digests/node-go.txt)
+            PRUNER_DIGEST=$(cut -d= -f2 digests/xmtp-prune-mls.txt)
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)